docs/user/guides/setup-single-sign-on-google-email.rst

```diff
@@ -0,0 +1,70 @@
+How to setup Single Sign-On (SSO) with Google Workspace
+=======================================================
+
+.. include:: /shared/admonition-rtd-business.rst
+
+This how-to guide will provide instructions on how to enable :abbr:`SSO (Single Sign-on)` with Google Workspace.
+If you want more information on this feature,
+please read :doc:`/commercial/single-sign-on`
+
+Prerequisites
+-------------
+
+Organization permissions
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+To change your Organization's settings,
+you need to be an *owner* of that organization.
+
+You can validate your ownership of the Organization with these steps:
+
+* Navigate to :guilabel:`Username dropdown` > :guilabel:`Organizations` > :guilabel:`<Organization name>`
```

Comment:
```suggestion
* Navigate to :guilabel:`<Username dropdown> 🔽️` > :guilabel:`Organizations` > :guilabel:`<Organization name>`
```

Reply:
Hrm, we're using this in a bunch of places, but I do think it's a good addition.

Reply:
I thought of the emoji as an experiment, I haven't seen how it looks. So in case you apply it, we can decide what to do after?

Reply:
I'll leave it off for now, but it looks reasonable:

![Screenshot 2022-12-13 at 2 03 47 PM](https://user-images.githubusercontent.com/25510/207453830-c7e3cba4-1026-4627-9a94-5c533f5bc178.png)